### ISSUE TITLE
Flag merging creates gcloud unrecognized arguments error

### ISSUE THREAD

**Issue by steveoh** [google-github-actions/deploy-cloudrun#342]
### TL;DR

I believe the concatenation of the flags etc is missing some spaces. Possibly I am using an unexpected yaml multiline string value which is causing this.

### Expected behavior

The action to deploy a revision

### Observed behavior

The gcloud command seems to be malformed causing a pipeline error

### Action YAML

```yaml
- name: 🚀 Deploy to Cloud Run
        id: deploy
        uses: google-github-actions/deploy-cloudrun@v0
        with:
          service: app
          image: gcr.io/${{ secrets.PROJECT_ID }}/app
          region: us-west3
          flags: |
            --service-account=...@***.iam.gserviceaccount.com
            --vpc-connector=***
            --vpc-egress=all-traffic
            --max-instances=1
            --concurrency=1
            --cpu=4
            --memory=6Gi
            --timeout=30m
          env_vars: |
            URLS=http://+:8080
          secrets: |
            /secrets/db/connection=connections:latest
            /secrets/email/key=key:latest
```

### Log output

```text
gcloud beta run deploy app --image gcr.io/***/app --quiet --platform managed --region us-west3 --update-env-vars URLS=http://+:8080 --update-secrets /secrets/db/connection=connections:latest,/secrets/email/key=key:latest --service-account ...@***.iam.gserviceaccount.com--vpc-connector *** --vpc-egress all-traffic --max-instances 1 --concurrency 1 --cpu 4 --memory 6Gi --timeout 60m --project *** --format json`: ERROR: (gcloud.beta.run.deploy) unrecognized arguments: ***
```

### Additional information

It appears that the service account and the --vpc-connector are not separated by a space? I tested the action step above successfully on `0.9.0` and it fails with the `0.10.0` release (allow for multi line flags ([#320](https://www.github.com/google-github-actions/deploy-cloudrun/issues/320)) ([c5ec47b](https://www.github.com/google-github-actions/deploy-cloudrun/commit/c5ec47bd62298b802bfff30783b634810a057e5b)))

**Comment by steveoh** [comment]
I checked out the project to try to test this behavior and the unit tests only ran 3 tests. And I don't see a way to test the gcloud command that is run. I think it could be nice to move the [cmd building](https://github.com/google-github-actions/deploy-cloudrun/blob/main/src/deploy-cloudrun.ts#L103-L222) to a function that could be tested. It's very possible I don't understand how these tests are organized though.

**Comment by sethvargo** [comment]
Duplicate of https://github.com/google-github-actions/deploy-cloudrun/issues/341
